docs/about/index.rst

```diff
@@ -19,14 +43,147 @@ This project follows the
 
 .. _roadmap:
 
-Project roadmap
----------------
+Project focus areas
+-------------------
+
+As an open-source project, Fairlearn strives to incorporate the best of
+research and practice.
+AI is a rapidly evolving field, and fairness in AI is all the more so.
+We therefore encourage researchers, practitioners, and other stakeholders to
+contribute fairness metrics and assessment tools, unfairness mitigation
+algorithms, case studies and other educational materials to Fairlearn as we
+experiment, learn, and evolve the project together.
+
+Below we list the key areas that we prioritize in the short
+and medium term, but we are happy to consider other directions
+if they are aligned with the mission of Fairlearn and there is enough commitment
+from the contributors. If you want to get involved, please
+:ref:`reach out <communication>`. For concrete opportunities and
+work in progress please review our
+`issues <https://github.com/fairlearn/fairlearn/issues>`_.
+
+Short and medium-term focus areas
```

Comment:
Should fairness mitigation algorithms be included as one of the focus areas?

Reply:
I think they should. We've gotten that as a feature request several times, too, and recently created a few issues accordingly, so it feels right to add it (even though it's perhaps not the top priority).